### PR TITLE
Added support for LinkingManager and Appearance modules with stub

### PR DESCRIPTION
**Pull request by munezbn**
## Description
Added stubbed modules  LinkingManager and Appearance with dummy events
Added test apps for both modules with event handling
Added UnimplementedTurboModule for few modules which are required for rn-tester launch

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| Linux   |    ✅     |
| iOS     |    ❌     |
| Android |    ❌     |
| Windows |    ❌     |

## Checklist

* [x] I have tested this on Ubuntu OS device
* [ ] I added the documentation in `README.md`
* [ ] I updated the typings files (`privateTypes.ts`, `types.ts`)
* [x] I added a sample use of the API (`modules/Appearence/SimpleAppearenceModuleTestApp.js`, `modules/LinkingManager/SimpleLinkingManagerTestApp.js`)

### CODE REVIEW

File: ReactSkia/core_modules/RSkAppearanceModule.cpp [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
```
Comment (chatrapathi06) [inline ReactSkia/core_modules/RSkAppearanceModule.cpp:1]: 2022 --> 2023 in the Copyright

File: ReactSkia/core_modules/RSkAppearanceModule.h [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
```
Comment (chatrapathi06) [inline ReactSkia/core_modules/RSkAppearanceModule.h]: 2022 --> 2023 in the Copyright

File: ReactSkia/core_modules/RSkLinkingManagerModule.cpp [hunk 3]
```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
```
Comment (chatrapathi06) [inline ReactSkia/core_modules/RSkLinkingManagerModule.cpp]: 2022 --> 2023 in the Copyright

File: ReactSkia/core_modules/RSkLinkingManagerModule.h [hunk 4]
```diff
@@ -0,0 +1,76 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
```
Comment (chatrapathi06) [inline ReactSkia/core_modules/RSkLinkingManagerModule.h]: 2022 --> 2023 in the Copyright

**Review by chatrapathi06 — APPROVED** [review]

**Review by shefayaparwin — APPROVED** [review]

**Pull request merged**